src/auth/ImplicitGrantStrategy.ts

```diff
@@ -52,7 +52,8 @@ export default class ImplicitGrantStrategy implements IAuthStrategy {
                 token_type: hashParams.get("token_type") ?? "",
                 expires_in: parseInt(hashParams.get("expires_in") ?? "0"),
                 refresh_token: hashParams.get("refresh_token") ?? "",
-                expires: Number(hashParams.get("expires")) || 0
+                expires: Number(hashParams.get("expires")) || 0,
+                state,
```

Comment:
❓ Not sure if I should use the state that's passed in as an arg here or use the query params like the other fields are using.... suggestions very welcom